examples/routing_service/routing_service_constrained_wan/README.md

```diff
@@ -0,0 +1,8 @@
+# Example: Routing Service Constrained WAN
```

Comment:
We should have a readme file that explains:
* the feature/scenario that this example shows
* how to build the example -- not applicable in this case
* how to run the example

Usually one readme at the same level of the 'language folders' explains the first point, and another one inside each programming language supported with the building and running information. As this is a special case, we can keep all these things in one single readme file.

eg: https://github.com/rticommunity/rticonnextdds-examples/tree/master/examples/connext_dds/asynchronous_publication

Reply:
let me know if the new readme is descriptive enough